src/sftp/handler_test.go

```diff
@@ -115,6 +236,79 @@ func Test_CopyFiles_SuccessfullyCopiesFiles(t *testing.T) {
 	assert.NotContains(t, buffer.String(), "Failed to read directory ")
 }
 
+func Test_Close_FailsToCloseSFTPClient(t *testing.T) {
+	mockBlobHandler := &mocks.MockBlobHandler{}
+	mockSftpClient := new(MockSftpWrapper)
+	mockCredentialGetter := new(mocks.MockCredentialGetter)
+	mockZipHandler := &MockZipHandler{}
+
+	mockSftpClient.On("Close", mock.Anything).Return(errors.New(""))
+
+	defaultLogger := slog.Default()
+	defer slog.SetDefault(defaultLogger)
+
+	buffer := &bytes.Buffer{}
+	slog.SetDefault(slog.New(slog.NewTextHandler(buffer, nil)))
+
+
+	sftpHandler := SftpHandler{sftpClient: mockSftpClient, blobHandler: mockBlobHandler, credentialGetter: mockCredentialGetter, zipHandler: mockZipHandler}
+
+	sftpHandler.Close()
+
+	assert.Contains(t, buffer.String(), "Failed to close SFTP client")
+}
+
+
+/*func Test_Close_FailsToCloseSshClient(t *testing.T) {
```

Comment:
Can we explain why this is commented out? Perhaps via a code comment

Reply:
Deleted duplicate test!